builder/builder_test.go

```diff
@@ -973,7 +928,7 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 			})
 
 			it("adds the env vars as files to the image", func() {
-				layerTar, err := baseImage.FindLayerWithPath("/platform/env/SOME_KEY")
+				layerTar, err := baseImage.FindLayerWithPath("/cnb/platform/env/SOME_KEY")
```

Comment:
Is this remanent of moving the platform and then reverting it? Shouldn't it be `/platform/env/SOME_KEY`?